ci/kubetest/conftest.py

```diff
@@ -30,7 +30,7 @@ def reset_k8s() -> None:
     #
     log.debug("Deleting all the namespaced resources...")
     namespaces_raw = exec_subprocess(
-        'kubectl get namespaces --no-headers -o custom-columns=":metadata.name" | grep -v "kube-" | grep -v "default" | grep -v "kubetest-" | tr "\n" ","'
+        'kubectl get namespaces --no-headers -o custom-columns=":metadata.name" | grep -v "kube-" | grep -v "local-path-storage" | grep -v "ingress-nginx" | grep -v "default" | grep -v "kubetest-" | tr "\n" ","'
```

Comment:
Non blocking comment: @hazzadous why do we need this?

Reply:
Purely for my kind setup in GitHub Codespaces. Otherwise we wipe out the local-storage PV provisioner, and the ingress I have deployed.